### PR TITLE
chore: add some logging and timings to analyze consensus processing perf

### PR DESCRIPTION
**Pull request by dpc**

### CODE REVIEW

File: fedimint-core/src/db/mod.rs [hunk 1]
```diff
@@ -235,22 +235,30 @@ impl Database {
...
-                        if max_attempts
-                            .map(|max_att| max_att <= curr_attempts)
-                            .unwrap_or(false)
-                        {
-                            return Err(AutocommitError::CommitFailed {
-                                attempts: curr_attempts,
-                                last_error: err,
-                            });
+                Ok(val) => {
+                    let _timing /* logs on drop */ = timing::TimeReporter::new("autocmmit - commit_tx");
```
Comment (m1sterc001guy) [inline fedimint-core/src/db/mod.rs:239]: Nit: typo "autocommit"

**Review by m1sterc001guy — APPROVED** [review]
LGTM

**Comment by m1sterc001guy** [comment]
Will defer merge decision if you wanna fix the typo or not :)

**Review by elsirion — APPROVED** [review]
ACK, feel free to fix he typo or merge @dpc. [`tracing_timing`](https://docs.rs/tracing-timing/latest/tracing_timing/) might also be interesting.

**Pull request merged**